AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -416,11 +426,6 @@ - (BOOL)startService:(Class)clazz
       [service updateConfigurationWithAppSecret:appSecret transmissionTargetToken:transmissionTargetToken];
     }
 
-    // Send start service log.
-    if (sendLog && fromApplication) {
```

Comment:
How is this related to the auth token storage initialization?  I'm not sure why this was removed.

Reply:
`sendLog` is called in startService**S**, the change is handle finalization logic only in one place.

The behavior of  `sendLog` isn't changed